packages/angular_devkit/build_angular/src/karma/index.ts

```diff
@@ -127,7 +127,9 @@ export function execute(
           return karmaStart.then(() => karmaServerWithStop.stop());
         }
       };
-    })),
+    }).pipe(
```

Comment:
This is already in a pipe call.  Should be able to put the default if empty in the same pipe.

Reply:
Good catch!